### ISSUE TITLE
BadInputException: Unhandled auth type: app, user

### ISSUE THREAD

**Issue by SamSchott** [dropbox/dropbox-sdk-python#369]
**Describe the bug**
Making `list_folder`, `list_folder/continue` and `list_folder/get_latest_cursor` API calls through the SDK raises a `BadInputException`:

```traceback
  File "/opt/homebrew/lib/python3.9/site-packages/dropbox/base.py", line 1976, in files_list_folder_continue
    r = self.request(
  File "/opt/homebrew/lib/python3.9/site-packages/dropbox/dropbox_client.py", line 321, in request
    res = self.request_json_string_with_retry(host,
  File "/opt/homebrew/lib/python3.9/site-packages/dropbox/dropbox_client.py", line 477, in request_json_string_with_retry
    return self.request_json_string(host,
  File "/opt/homebrew/lib/python3.9/site-packages/dropbox/dropbox_client.py", line 564, in request_json_string
    raise BadInputException('Unhandled auth type: {}'.format(auth_type))
dropbox.dropbox_client.BadInputException: Unhandled auth type: app, user
```

Apparently the auth type "app, user" which is specified by the SDK is invalid. Just "app" or "user" or both accepted as valid auth types. This appears to have been introduced with #368  in v11.13.

Looking at the code which actually makes the request, "app, user" is in fact **not** an accepted auth type in the SDK:

https://github.com/dropbox/dropbox-sdk-python/blob/4ed82373d161ca68c559ec9308dc6c392dd51651/dropbox/dropbox_client.py#L63-L66

https://github.com/dropbox/dropbox-sdk-python/blob/4ed82373d161ca68c559ec9308dc6c392dd51651/dropbox/dropbox_client.py#L547-L564

**To Reproduce**
Make one of the above listed API calls

**Versions**
* What version of the SDK are you using? 11.13
* What version of the language are you using? Python 3.9
* What platform are you using? macOS 11

**Comment by greg-db** [comment]
Thanks for the report! We'll look into it. Please downgrade to v11.12.0 for now.

**Comment by greg-db** [comment]
This is fixed in v11.13.1 now.

**Comment by SamSchott** [comment]
Excellent, thank you for the quick fix!
